### ISSUE TITLE
Screen flicker when Terminal is opened on top of Skype window

### ISSUE THREAD

**Issue by 13xforever** [microsoft/terminal#4528]
# Environment

```none
Windows build number: 10.0.18363 (Windows 10 1909)
Windows Terminal version (if applicable): 0.8.10261.0 (latest Windows Store release)
Built-in Skype app: 14.56.102.0
GPU: Intel UHD 620 (desktop skylake) using drivers 100.7463, running dual-monitor setup at 1080p each
```

# Steps to reproduce
Open Skype and maximize the window, open Windows Terminal with any profile that is using acrylic effect (I have it set at 0.75 for cmd, for example), observe flicker on repaint.

You can also have other legacy windows between skype and terminal (placing some modern app in-between can stop this issue)

# Expected behavior
Repaint is unnoticeable

# Actual behavior
Screen flickers

**Comment by DHowett-MSFT** [comment]
Really! If you place Calculator on top of Skype (calculator has a lot of acrylic) without Terminal, does it also flicker?

**Comment by 13xforever** [comment]
Calculator seems to be unaffected, and I haven't noticed any flicker in other apps before for that matter.

Not sure if this would be helpful, but here's a [bad video](https://1drv.ms/u/s!AruI8iDXabVJ3g6tlSWHXQo1VME0) that demonstrates the issue

**Comment by DHowett-MSFT** [comment]
Well yep, that's rather bad. Thanks for the report!

**Comment by DHowett-MSFT** [comment]
I'm concerned this might be related to the use of acrylic in xaml islands. Maybe #778 will make this less bad as we'll be redrawing the entire screen less?

**Comment by Priddlesaur** [comment]
I'm experiencing a very similar issue which I believe is related to this one. 

When I have the Terminal open and overlap it on top of Windows Media Player, it will flicker just like this, but only when the media player controls fade in and out.

I'm running a dedicated Nvidia GPU and an AMD CPU without integrated graphics.

**Comment by 13xforever** [comment]
I was unable to reproduce this on an Intel UHD Graphics 620 (mobile kaby lake-r) with the same drivers (100.7262-100.7755) on Windows 10 2004 (10.0.19041) wuith a 4K screen.

**Comment by vslynko** [comment]
I observe the same problem:
- Screen flickering is always happening when terminal is in focus
- No matter whether acrylic effect is enabled or not
- GeForce GTX 1080 with driver 442.50 (but also previous version)
- On light backgrounds the flickering is hard to notice, but still possible. On dark backgrounds it's very annoying
- No matter what other apps are on the screen - VS2019, Netflix, vscode etc, flickering happens only when terminal is in focus.
- Windows 10 64bit latest updates as of time of writing

But thanks for doing a great job.

**Comment by davefp** [comment]
@vslynko this is the exact problem I'm having. Glad it's not just me!

**Comment by davefp** [comment]
@vslynko I had success turning off G-Sync for Terminal as described in #649, perhaps that will be useful for you too.

**Comment by vslynko** [comment]
Yes, turning G-Sync off helps. Some issue and some solution also present for Plex.

**Comment by Ares9323** [comment]
Solved turning G-Sync off

GPU: RTX2070 Super - Driver 451.77

**Comment by vadviktor** [comment]
Why turning off G-Sync is a solution? I don't want to go in every time I want to play or want to code into Nvidia's panel to flip switches.

**Comment by vslynko** [comment]
@vadviktor - it was suggested to turn off G-sync only for terminal process, as a workaround.
